### PR TITLE
Week 18: Mongodb + express

### PR DESCRIPTION
**Pull request by monkeybusiness101**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 node_modules
 .DS_Store
+.env
```
Comment (karanmann) [inline .gitignore:3]: Cool that you decided to use .env

File: server.js [hunk 2]
```diff
@@ -2,17 +2,13 @@ import express from 'express'
...
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
+dotenv.config()
```
Comment (karanmann) [inline server.js:7]: Same here 👍

File: server.js [hunk 3]
```diff
@@ -27,9 +23,43 @@ const app = express()
...
+    averagePrice: Number,
+    totalVolume: Number,
+    totalBagsSold: Number,
+    smallBagsSold: Number,
+    largeBagsSold: Number,
+    xLargeBagsSold: Number,
+    region: String
+})
+
+if (process.env.RESET_DATABASE) {
```
Comment (karanmann) [inline server.js:38]: Nice :) 🥇

File: server.js [hunk 4]
```diff
@@ -27,9 +23,43 @@ const app = express()
...
+
+app.get('/', async (req, res) => {
+  res.send("We are advocates of avocados")
+})
+
+app.get('/avocados/id/:id', async (req, res) => {
+  const singleAvocado = await Avocado.findOne({ id: req.params.id })
+  res.json(singleAvocado)
+
 })
```
Comment (karanmann) [inline server.js:63]: Hej Max,
You code was really clean and easy to read. I would recommend to write a more detailed readme defining the endpoints it just gives it a clear documentation. 

But nevertheless great job 🥇

**Review by karanmann — APPROVED** [review]

File: server.js [hunk 5]
```diff
@@ -27,9 +23,43 @@ const app = express()
...
+app.get('/avocados', async (req, res) => {
+  const allAvocados = await Avocado.find()
+  res.json(allAvocados)
+})
+
+app.get('/', async (req, res) => {
+  res.send("We are advocates of avocados")
+})
+
+app.get('/avocados/id/:id', async (req, res) => {
```
Comment (KarolCislowski) [inline server.js:59]: `id` is a default key for your document so you can use `/avocados/:id` as an endpoint URL

**Review by KarolCislowski — APPROVED** [review]
Nice Project! 👍🏻

**Pull request closed without merging**